includes/options/class-amp-options-menu.php

```diff
@@ -143,112 +158,216 @@ public function add_menu_items() {
+							esc_url( 'https://amp.dev/about/websites' )
+						)
+					);
+					?>
+				</dd>
+				<dt>
+					<input type="checkbox" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[experiences][]' ); ?>" id="stories_experience" value="<?php echo esc_attr( AMP_Options_Manager::STORIES_EXPERIENCE ); ?>" <?php disabled( ! $has_required_block_capabilities ); ?> <?php checked( in_array( AMP_Options_Manager::STORIES_EXPERIENCE, $experiences, true ) ); ?>>
+					<label for="stories_experience">
+						<strong><?php esc_html_e( 'Stories', 'amp' ); ?></strong>
+					</label>
+				</dt>
+				<dd>
+					<?php if ( ! $has_required_block_capabilities ) : ?>
+						<div class="notice notice-info notice-alt inline">
+							<p>
+								<?php
+								$gutenberg = 'Gutenberg';
+								// Link to Gutenberg plugin installation if eligible.
+								if ( current_user_can( 'install_plugins' ) ) {
+									$gutenberg = '<a href="' . esc_url( add_query_arg( 'tab', 'beta', admin_url( 'plugin-install.php' ) ) ) . '">' . $gutenberg . '</a>';
+								}
+								printf(
+									/* translators: %s: Gutenberg plugin name */
+									esc_html__( 'To use stories, you currently must have the latest version of the %s plugin installed and activated.', 'amp' ),
+									$gutenberg // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+								);
+								?>
+							</p>
+						</div>
+					<?php endif; ?>
+					<?php
+					echo wp_kses_post(
+						sprintf(
+							/* translators: %s: Stories documentation URL. */
+							__( 'Stories is a visual storytelling format for the open web which immerses your readers in fast-loading, full-screen, and visually rich experiences. Stories can be a great addition to your overall content strategy. Read more about <a href="%s" target="_blank">AMP Stories</a>.', 'amp' ),
+							esc_url( 'https://amp.dev/about/stories' )
+						)
+					);
+					?>
+				</dd>
+			</dl>
+			<script>
+				/*
+				 * Toggle visibility of setting sections based on whether or not their respective experiences are enabled.
+				 * Ensure that at least one experience is selected, either Website, Stories, or both.
+				 */
+				( function( $, optionInputName, mustSelectMessage ) {
+					const websiteExperienceInput = $( '#website_experience' )[0];
+					const checkboxInputs = $( 'input[name="' + optionInputName + '"]' );
+
+					const handleExperiencesUpdate = () => {
+						const checkedCount = checkboxInputs.filter( ':checked' ).length;
+						if ( 0 === checkedCount ) {
+							websiteExperienceInput.setCustomValidity( mustSelectMessage );
+						} else {
+							websiteExperienceInput.setCustomValidity( '' );
+						}
+
+						checkboxInputs.each( function() {
+							document.body.classList.toggle( 'amp-experience-' + this.value, this.checked );
+						} );
+					};
+
+					checkboxInputs.on( 'change', handleExperiencesUpdate );
+					handleExperiencesUpdate();
```

Comment:
It's nice how the 'Website' options only display when 'Website' is checked:

![website-mode](https://user-images.githubusercontent.com/4063887/59235272-fce94c00-8bb5-11e9-863f-c17eb9dc7a76.gif)